docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,71 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Reduce number of requests from coordinator to workers ({issue}`8460`)
+* Improve query performance by sending collected dynamic filters from coordinator to workers. ({issue}`5183`)
+* Improve performance of inequality joins where join condition sides have different type. ({issue}`8500`)
+* Fix query failure when there is a constant predicate on some
+  column `col` (e.g `col=1`), followed by  `ORDER BY col` and `LIMIT`. ({issue}`8535`)
+* Fix SHOW CREATE SCHEMA. ({issue}`8598`)
+* Fix query failure when running `SHOW CREATE SCHEMA information_schema`. ({issue}`8600`)
+* Improve performance of queries with `WHERE` predicates. ({issue}`8624`)
+* Disallow `DROP SCHEMA` if the schema is not empty. ({issue}`8660`)
+  
+## Security
+
+* Add support for automatic configuration of TLS for {doc}`/security/internal-communication`. This removes the need to provision
+  per-worker TLS certificates. ({issue}`7954`)
+  
+## CLI
+
+* Fix auto completion when pressing the tab button. ({issue}`8529`)
+
+## Hive connector
+
+* Ensure generation of unique ROW IDs during UPDATE on ORC ACID transactional tables.
+  Before it was possible that Hive was not able to read the table updated by Trino ({issue}`8268`)
+* Ensure generation of unique ROW IDs during INSERT into ORC ACID transactional tables.
+  Before it was possible that subsequent DELETE from a table would delete more rows than intended ({issue}`8452`)
+  
+## Iceberg connector
+
+* Add support for Trino views. ({issue}`8540`)
+* Fix query failure when joining with a bucketed Iceberg table. ({issue}`7502`)
+* Fix query failure when showing stats for a bucketed Iceberg table. ({issue}`8616`)
+* Fix query failure when joining with a partitioned table that has structural columns (`array`, `map` or `row`). ({issue}`8647`)
+* Fail queries on tables with row-level deletes, since it is not supported. ({issue}`8450`)
+* Fix failures for queries that write tables in Parquet format. ({issue}`5201`)
+
+## MongoDB connector
+
+* Add support for `json` type. ({issue}`8352`)
+* Support reading MongoDB `DBRef` type. ({issue}`3134`)
+
+## Phoenix connector
+
+* Improve performance of `ORDER BY ... LIMIT` queries on sorted data for Phoenix 5. ({issue}`8171`)
+
+## Other connectors
+                                           
+This change applies to the ClickHouse, Druid, MemSQL, MySQL, Oracle, Phoenix, PostgreSQL, Redshift and SQLServer connectors:
+* Add `write.batch-size` connector configuration to control JDBC batch size used during writes.
+  It can also be controlled using the `write_batch_size` session property. ({issue}`8434`)
+
+This change applies to the MemSQL, MySQL, Oracle, Phoenix, PostgreSQL, Redshift and SQLServer connectors:
```

Comment:
unroll

Reply:
I'll do that after we settle on the language for the entries. I also want to verify that the list of connectors for each of these entries is complete. @hashhar @kokosing